deepchem/models/torch_models/layers.py

```diff
@@ -3743,3 +3743,71 @@ def forward(
             P = pair_features
 
         return [A, P]
+
+class Highway(nn.Module):
```

Comment:
You will need to add this class to the docs/ folder. See where the other layers are and add to the same rst file

Reply:
Sure, once the formatting and testing is done then I'll add

Reply:
Make sure to also add it to torch_layers.csv in addition to the layers.rst file in the docs. Please add it before next review

Reply:
Yes, added